index.html

```diff
@@ -3,15 +3,186 @@
+             </div>
+              </div>
+              <div class="speakers">
+             <img src="images/speaker_01 1 (2).png" alt="speaker">  
+             <div class="text-align">
+              <h4>Lila tretikov</h4>
+              <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+              <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+             </div>
+            </div> 
+        </article>
+        <article>
+            <div class="speakers">
+                <img src="images/speaker_01 1 (3).png" alt="speaker">  
+                <div class="text-align">
+                 <h4>Ryan Merkley</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+               <div class="speakers">
+                <img src="images/speak.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>Kilnam Chon</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div>
+            </article> 
+            <article>
+               <div class="speakers">
+                <img src="images/speaker_01 1.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>Yochai Benkler</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+            
+               <div class="speakers">
+                <img src="images/spea.png" alt="speaker">  
+                <div class="text-align">
+                 <h4>SohYeong Noh</h4>
+                 <h6>Berkman Professor of Entrepreneurial Legal Studies at Harvard Law School</h6>
+                 <p class="para">Benkler studies commons-based peer production, and published his seminal book, The Wealth of Networks in 2006</p>
+                </div>
+               </div> 
+        </article> -->
+        <button id="btn-more" class="btn-expand" type="button">
+            MORE <img src="images/arrow_down 1.png" alt="" id="down">
+          </button>
+          <button id="btn-hide"  class="btn-expand" type="button">
+            HIDE <i class="fa-solid fa-chevron-up" id="up"></i>
+          </button>
+      </section>
```

Comment:
- [ ] As stated in the project requirement Kindly make sure that **The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable**. So Kindly Adjust the overlapping between the image and text , make the background of the speaker section just like the original Design and some of the the speakers position is not visible Kindly Fix the Issues. As front end developer Kindly I recommend you to give attention for the minor details it helps you to attract the clients for the future projects. 
 - Your Speaker Section Look likes this : -
 
![speaker](https://github.com/jerymey/Capstone/assets/93598189/0cc38580-7063-47ec-8b60-aff2e0626743)

- The Original Speaker section Look likes this : 
![spe](https://github.com/jerymey/Capstone/assets/93598189/2187d316-dd5e-45c5-9e7f-fe145d46b2f6)